packages/apps/desktop-viewer-test/src/frontend/index.tsx

```diff
@@ -22,7 +23,15 @@ const viewerFrontendMain = async () => {
   document.documentElement.classList.add(`iui-theme-dark`);
 
   // when initialization is complete, render
-  ReactDOM.render(<App />, document.getElementById("root"));
+  const container = document.getElementById("root") as HTMLElement;
+  const root = createRoot(container);
+  UiFramework.childWindows.useCreateRoot(createRoot);
```

Comment:
@raplemie does this need to be done after UiFramework is initialized?

Reply:
It doesnt matter, just needs to be done before a first child window is created, so it could be done on line 3 of your index and it would work fine.